src/main/java/potaton/Potaton.java

```diff
@@ -0,0 +1,299 @@
+package potaton;
+
+import potaton.task.Deadline;
+import potaton.task.Events;
+import potaton.task.Task;
+import potaton.task.ToDos;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Potaton {
```

Comment:
You have little or no [javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments). Including them in this class would greatly help the reader quickly understand what some of the methods are doing.